SampleScenarios/Scenarios/MarkerSample/Scripts/MarkerSampleInstructions.cs

```diff
@@ -14,7 +14,8 @@ private void Start()
             m_text = @"Sample: ARMarker
 
 - Use the 'Change Transform Mode' button to alter the center of the marker between origin and geometric center.
-- Use the 'Change Default Transform Mode' button to change the defaul transform mode of newly detected markers.";
+- Use the 'Change Default Transform Mode' button to change the default transform mode of newly detected markers.
+- A blue cube indicates a marker thats being tracked. A grey cube indicates a marker with limited tracking.";
```

Comment:
nit: "thats being tracked" -> "that's being tracked"
Thanks for switching the color here too!

Reply:
Addressed